src/Dumplie/Application/Query/Customer/Result/CartItem.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Query\Customer\Result;
+
+final class CartItem
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var float
+     */
+    private $price;
+
+    /**
+     * @var string
+     */
+    private $currency;
+
+    /**
+     * CartItem constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param float $price
+     * @param string $currency
+     */
+    public function __construct(string $sku, int $quantity, float $price, string $currency)
+    {
+        $this->sku = $sku;
+        $this->quantity = $quantity;
+        $this->price = $price;
```

Comment:
Czy nie byłoby lepiej aby price, currency, tax były VO? Po dodaniu do koszyka item nie powinien mieć tych wartości już zmienianych, ale to zależy od strategii danego sklepu.